lib/PageInteractions.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package PageInteractions;
+use strict;
+use warnings FATAL => 'all';
+
+use testapi;
+use parent 'Exporter';
+use Exporter;
+
+our @EXPORT = qw(
+    find
+    send_keys
+    fill_in_password
+    is_displayed
+);
+
+sub find {
```

Comment:
Creating new function which will do the same as old function is nonsense
I like that you want to make some thing more obvious, but this "wrappers" make openQA just more complicated why don't you add this feature directly into send_key, type_password, check_screen and so on ?

Reply:
Thanks for the comment. Yes, you're right, but I just wanted to show what I would like to have in openQA and added just dump wrappers to get the feedback about the whole idea of having every step somehow logged on openQA job overview page.

I totally agree that it should be implemented in os-autoinst and openQA should also allow to show the steps in more user-friendly and clear way, rather then using 'record_info' squares for each step.